app/src/main/res/values/styles.xml

```diff
@@ -44,6 +44,8 @@
         <item name="pause">@drawable/ic_pause_black_24dp</item>
         <item name="settings">@drawable/ic_settings_black_24dp</item>
         <item name="ic_hot">@drawable/ic_whatshot_black_24dp</item>
+        <item name="ic_kiosk_local">@drawable/ic_kiosklocal_black_24dp</item>
```

Comment:
Please insert an underscore after `kiosk` in the file names to increase readability.